### PR TITLE
Job examples cleanup

### PR DESCRIPTION
**Pull request by mimowo**
- move the job examples down under kubernetes/
- delete the unnecessary reference to build_and_push.sh script
- cleanup run.sh & clean.sh scripts

### CODE REVIEW

File: kubernetes/job-examples/ml_training_jax/clean.sh [hunk 1]
```diff
@@ -5,3 +5,4 @@ kubectl delete -f service.yaml --ignore-not-found=true
 kubectl delete -f job.yaml --ignore-not-found=true
 kubectl delete pods -l job-name=myjob --ignore-not-found=true --force --grace-period=0
 kubectl wait --for=delete pod -l job-name=myjob --timeout=60s
+kubectl delete configmap script --ignore-not-found=true
```
Comment (alculquicondor) [inline kubernetes/job-examples/ml_training_jax/clean.sh]: new line

File: kubernetes/job-examples/ml_training_pytorch/clean.sh [hunk 2]
```diff
@@ -5,3 +5,4 @@ kubectl delete -f service.yaml
 kubectl delete -f job.yaml --ignore-not-found=true
 kubectl delete pods -l job-name=myjob --ignore-not-found=true --force --grace-period=0
 kubectl wait --for=delete pod -l job-name=myjob --timeout=60s
+kubectl delete configmap script --ignore-not-found=true
```
Comment (alculquicondor) [inline kubernetes/job-examples/ml_training_pytorch/clean.sh]: new line

**Review by alculquicondor — APPROVED** [review]

File: kubernetes/job-examples/ml_training_jax/clean.sh (hunk 1 above)
Comment (mimowo) [reply kubernetes/job-examples/ml_training_jax/clean.sh]: done

File: kubernetes/job-examples/ml_training_pytorch/clean.sh (hunk 2 above)
Comment (mimowo) [reply kubernetes/job-examples/ml_training_pytorch/clean.sh]: done

**Comment by mimowo** [comment]
@alculquicondor  thanks for reviewing. I will merge it and will open another follow up if something pops up.

**Pull request merged**